src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -428,4 +438,122 @@ public synchronized File getNextMixinMappings() {
 	public List<LoomDecompiler> getDecompilers() {
 		return decompilers;
 	}
+
+	public void runs(Closure<?> conf) {
+		runs.configure(conf);
+	}
+
+	// Workaround for ugly closures in Kotlin DSL buildscripts
+	public void run(String name, Action<RunConfigSettings> settingsAction) {
+		runs.create(name, settingsAction);
+	}
+
+	public NamedDomainObjectContainer<RunConfigSettings> getRuns() {
+		return runs;
+	}
+
+	public static class RunConfigSettings implements Named {
+		private final List<String> vmArgs = new ArrayList<>();
+		private final List<String> programArgs = new ArrayList<>();
+		private String mode;
```

Comment:
Some basic comments might be nice on this, `mode` wasnt that clear what it does.